test/e2e/GuardianVerifier.t.sol

```diff
@@ -78,6 +78,12 @@ contract OwnableValidatorRecovery_AbstractedRecoveryModule_Test is
     // End to end test
     function test_Recover_RotatesOwnerSuccessfully() public {
         skipIfCommandHandlerType(CommandHandlerType.SafeRecoveryCommandHandler);
+        if (
+            accountAddress1 !=
```

Comment:
This check is hard to understand at first glance, could `0x0929E9d9bC617836B650Ee4E419f52D31831C806` be assigned to a variable? And assume we are just enforcing that the expected address is correct? Should we revert instead of skip - or let me know if I am misinterpreting the check

Reply:
Actually I have added this check, as the only account this will pass is the one in argument, it's due to the because of the specific zk-email proof we generated and use to verify.

I can perhaps assign it to a variable if that's required.

Reply:
Does it need to be a check? Can we just assume that the test is set up for a specific account? I'm not sure what the check is preventing if the test setup always uses this account

Reply:
yeah, we don't need to check for that, we have the same account salt so it'll be the same address in the setup

Reply:
Added a note just for future reference

Reply:
Actually just remembered the CI, also runs test with other type of accounts like safe, etc. which fails the test as they will generate a different address even with the same salt.

Should I add it back in, or add an account type skip check ?

Reply:
> Should I add it back in, or add an account type skip check ?

Right, yeah let's add it back in and leave a comment explaining why it is needed (also might as well put the address in a variable for readability)

Reply:
Done!